src/FrontMatterConverter.ts

```
@@ -1,187 +1,202 @@
 import { ObsidianRegex } from './core/ObsidianRegex';
-import { Converter } from './core/Converter';
+import { pipe } from './core/fp';
 import yaml from 'js-yaml';
+import {
+  ConversionError,
+  ConversionResult,
+  Either,
+  FrontMatter,
+  chain,
```

ESLint

[error] 'chain' is defined but never used.

(@typescript-eslint/no-unused-vars)